src/test/java/com/training/utilities/tests/GetJson.java

```diff
@@ -1,23 +1,13 @@
 package com.training.utilities.tests;
 
-import com.training.utilities.GetJsonValue;
-import org.junit.jupiter.api.BeforeEach;
+import com.training.base.BaseTest;
 import org.junit.jupiter.api.Test;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import java.util.Map;
 
-public class GetJson {
+public class GetJson extends BaseTest {
```

Comment:
Every Class test should have a suffix 'Test'
Eg : GetJsonTest, GetDataTest ....